src/foam/java/refinements.js

```diff
@@ -556,6 +556,14 @@ foam.LIB({
               return `get${foam.String.capitalize(p.name)}();`
             }).join('\n')
         });
+
+        cls.method({
+          name: 'equals',
+          type: 'boolean',
+          args: [ { name: 'o', type: 'Object' } ],
+          visibility: 'public',
+          body: `return super.equals(o);`
```

Comment:
I don't like this solution. Better to just re-insert super code here to be sure it's correct.

Reply:
This is just cheating.

Reply:
No, just copy the code out of the super class and put it here in body: